data/IL/incentives.json

```diff
@@ -0,0 +1,1110 @@
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_stove",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 100
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$100 rebate for Induction Cooktop."
+    }
+  },
+  {
+    "id": "IL-4",
+    "authority_type": "utility",
+    "authority": "il-commonwealth-edison",
+    "payment_methods": [
+      "pos_rebate",
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 off ENERGY STAR certified Smart Thermostats."
+    }
+  },
+  {
+    "id": "IL-5",
+    "authority_type": "utility",
+    "authority": "il-commonwealth-edison",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "weatherization",
+    "program": "il_commonwealthEdison_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $2 off on Weatherization Products such as caulking, weatherstripping, insulation, and door sweeps/seals."
+    }
+  },
+  {
+    "id": "IL-6",
+    "authority_type": "utility",
+    "authority": "il-corn-belt-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "il_cornBeltEnergyCorporation_rebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 750
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$750 rebate for replacing existing geothermal with new geothermal system with an electric backup."
+    }
+  },
+  {
+    "id": "IL-7",
+    "authority_type": "utility",
+    "authority": "il-corn-belt-energy",
+    "payment_methods": [
+      "unknown"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "il_cornBeltEnergyCorporation_rebateProgram",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1100,
+      "unit": "ton"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $1100/ton with $6/ton or $7/ton monthly fee for GeoCents Geothermal program."
+    }
+  },
+  {
+    "id": "IL-8",
+    "authority_type": "utility",
+    "authority": "il-corn-belt-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "il_cornBeltEnergyCorporation_rebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Up to $2,000 rebate for geothermal heat pumps."
```

Comment:
Some of these descriptions are a little too terse, IMO. Feel free to look at other states for examples and push back if I'm wrong, but in general we like to include some details about important restricts on e.g. SEER, max/min size, etc. if they are available (if not, don't worry about it).

Reply:
Our maximum is 150 – you don't have to hit that, but I'd review anything below ~50 or so.

Reply:
Went through and gave the short strings some more love.